commands/remove.go

```diff
@@ -0,0 +1,45 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/cristianoliveira/ergo/proxy"
+)
+
+// RemoveService removes a service from the configuration and tells the proxy to
+// remove it from the configFile
+func RemoveService(config *proxy.Config, service proxy.Service, configFile string) {
+
+	var oldService *proxy.Service
+	for _, srv := range config.Services {
+		if srv.URL == service.URL || srv.Name == service.Name {
+			oldService = &srv
+		}
+	}
+
+	if oldService == nil {
+		fmt.Printf("Service %s not found\n", service.Name)
+		return
+	}
+
+	services := []proxy.Service{}
+	for _, srv := range config.Services {
+		if srv.Name != service.Name || srv.URL == service.URL {
+			services = append(services, srv)
+		}
+	}
+
+	if len(services) > len(config.Services) {
+		fmt.Printf("Failed to remove service %s", service.Name)
+		return
+	}
+
+	config.Services = services
+	err := proxy.RemoveService(configFile, *oldService)
```

Comment:
I liked the way you split this logic :)

Reply:
Thanks, I was thinking about how to best remove the service given that it exists in two different places.